Detailed HVAC setpoints (GEB), try 2 (#1056)
NREL/OpenStudio-HPXML: docs/source/workflow_inputs.rst

Detailed schedule inputs are provided via CSV file(s) that should be referenced in the HPXML file as a ``/HPXML/SoftwareInfo/extension/SchedulesFilePath``.

> Occupancy related schedule columns must be normalized to MAX=1; that is, these schedules only define *when* energy is used, not *how much* energy is used.
> Thermostat setpoint schedule columns do not have these constraints.

Reviewer: Thinking ahead to the WH GEB changes, I suggest creating a new column below called 'Units', which is either `frac` or `F`, and then making this change:

```suggestion
Columns with units of `frac` must be normalized to MAX=1; that is, these schedules only define *when* energy is used, not *how much* energy is used.
```
